airbyte-webapp/src/components/ui/Switch/Switch.tsx

```diff
@@ -12,6 +12,7 @@ interface SwitchProps extends Omit<React.InputHTMLAttributes<HTMLInputElement>,
   loading?: boolean;
   size?: SwitchSize;
   variant?: SwitchVariant;
+  testId?: string;
```

Comment:
Because this uses `React.InputHTMLAttributes<HTMLInputElement>` it already supports `data-testid` as a prop

Reply:
fixed